agent/src/main/java/com/appland/appmap/config/AppMapConfig.java

```diff
@@ -71,6 +105,10 @@ public static AppMapConfig load(File configFile) {
     return singleton;
   }
 
+  public static AppMapConfig load(File configFile) {
```

Comment:
Is this overload used anywhere except for tests? Perhaps it could be removed if that's the case.